easybuild/easyconfigs/l/libecpint/libecpint-1.0.7-foss-2021a.eb

```
@@ -0,0 +1,32 @@
+easyblock = 'CMakeMake'
+
+# easyconfig file for libecpint
+name = 'libecpint'
+version = '1.0.7'
+
+homepage = 'https://github.com/robashaw/libecpint'
+description = """Libecpint is a C++ library for the efficient evaluation of integrals over ab initio effective core potentials,
+using a mixture of generated, recursive code and Gauss-Chebyshev quadrature."""
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+toolchainopts = {'cstd': 'c++11'}
+
+source_urls = ['https://github.com/robashaw/libecpint/archive']
+sources = ['v%(version)s.tar.gz']
+checksums = ['e9c60fddb2614f113ab59ec620799d961db73979845e6e637c4a6fb72aee51cc']
+
+builddependencies = [('CMake', '3.20.1')]
```

@benczaja this also needs googletest builddep:
```suggestion
builddependencies = [
    ('CMake', '3.20.1'),
    ('googletest', '1.11.0'),
]
```